mne/viz/topomap.py

```diff
@@ -92,6 +94,46 @@ def _prepare_topomap_plot(inst, ch_type, layout=None, sphere=None):
         picks, _ = _pair_grad_sensors(info, layout)
         pos = _find_topomap_coords(info, picks[::2], sphere=sphere)
         merge_grads = True
+    elif ch_type in ['hbo', 'hbr', 'fnirs_raw', 'fnirs_od']:
+        # Special case for merging nirs channels
+        picks = pick_types(info, meg=False, ref_meg=False, fnirs=ch_type,
+                           exclude='bads')
+        chs = [info['chs'][i] for i in picks]
```

Comment:
This is enough extra lines that it should probably all live in a `picks, pos, merge_grads = _average_fnirs_overlaps(...)` call or so. It makes the code more readable, and clearer what values we will use in the rest of this function (in my example, `picks`, `pos`, and `merge_grads`), what's required to compute the overlaps, etc.